spec/consensus/proposer-based-timestamp/pbts-sysmodel_002_draft.md

```diff
@@ -0,0 +1,222 @@
+# Proposer-Based Time v2 - Part I
+
+## System Model
+
+#### **[PBTS-CLOCK-NEWTON.0]**
+
+There is a reference Newtonian real-time `t` (UTC).
+
+No process has direct access to this reference time, used only for specification purposes.
+
+### Synchronized clocks
+
+Processes are assumed to be equipped with synchronized clocks.
+
+This requires processes to periodically synchronize their local clocks with an
+external and trusted source of the time (e.g. NTP servers).
+Each synchronization cycle aligns the process local clock with the external
+source of time, making it a *fairly accurate* source of real time.
+The periodic (re)synchronization aims to correct the *drift* of local clocks,
+which tend to pace slightly faster or slower than the real time.
+
+To avoid an excessive level detail in the parameters and guarantees of
+synchronized clocks, we adopt a single system parameter `PRECISION` to 
+encapsulate the potential inaccuracy of the synchronization mechanisms,
+and drifts of local clocks from real time.
+
+#### **[PBTS-CLOCK-PRECISION.0]**
+
+There exists a system parameter `PRECISION`, such that
+for any two processes `p` and `q`, with local clocks `C_p` and `C_q`,
+that read their local clocks at the same real-time `t`,  we have:
+
+- If `p` and `q` are equipped with synchronized clocks, then `|C_p(t) - C_q(t)| < PRECISION`
+
+`PRECISION` thus bounds the difference on the times simultaneously read by processes
+from their local clocks, so that their clocks can be considered synchronized.
+
+#### Accuracy
+
+The [previous system model][v1] included a second clock-related parameter, `ACCURACY`,
+that relates the values read by processes from their synchronized clocks with the real time.
```

Comment:
I think that this implies a relationship between the synchronized clock (ntp in this case) and the 'real' time. Should we make that somewhat explicit in this document?

Reply:
Accuracy is between clock readings and real time. We try to improve accuracy of the local clock by synchronizing it with a more accurate clock. By synchronizing here we mean trying to reduce (so, to improve) at most the precision between the local clock and the reference clock. As a result, our local clock accuracy will be the accuracy of the reference clock plus (so, reduced by) the precision achieved in the synchronization process.